pkg/parser/parser.y

```diff
@@ -14721,6 +14734,27 @@ OptionalShardColumn:
 		$$ = $2.(*ast.ColumnName)
 	}
 
+/********************************************************************
+ * OptimizeTableStmt
+ *
+ * OPTIMIZE [NO_WRITE_TO_BINLOG | LOCAL]
+ *     TABLE tbl_name [, tbl_name] ...
+ *******************************************************************/
+OptimizeTableStmt:
+	"OPTIMIZE" NoWriteToBinLogOpt LocalOpt "TABLE" TableNameList
```

Comment:
It will support stmt like `OPTIMIZE NO_WRITE_TO_BINLOG LOCAL TABLE t;` which is not allowed in MySQL.

Reply:
@ub-3 you could just follow the [yacc definition in MySQL](https://github.com/mysql/mysql-server/blob/1bfe02bdad6604d54913c62614bde57a055c8332/sql/sql_yacc.yy#L9658-L9669):

> opt_no_write_to_binlog:
>          /* empty */ { $$= 0; }
>        | NO_WRITE_TO_BINLOG { $$= 1; }
>        | LOCAL_SYM { $$= 1; }